sdk/ahnlich-client-py/README.md

```diff
@@ -0,0 +1,386 @@
+)
+```
+
+### Drop Index
+```py
+from ahnlich_client_py import AhnlichProtocol, AhnlichProtocol
+pro = protocol.AhnlichProtocol(address="127.0.0.1", port=1369)
+client = client.AhnlichProtocol(client=pro)
+
+response = client.drop_index(
+    store_name = "test store",
+    predicates=["job"],
+    error_if_not_exists=True
+)
+```
+
+
+### Set
+```py
+from ahnlich_client_py import AhnlichProtocol, AhnlichProtocol
+from libs import create_store_key
+pro = protocol.AhnlichProtocol(address="127.0.0.1", port=1369)
+client = client.AhnlichProtocol(client=pro)
+
+store_key = create_store_key(data=[5.0, 3.0, 4.0, 3.9, 4.9])
+store_value =  {"rank": "chunin"}
+
+
+response = client.set(
+    store_name = "test store",
+    inputs=[(store_key, store_value)]
+)
+```
+
+### Delete Key
+```py
+from ahnlich_client_py import AhnlichProtocol, AhnlichProtocol
+from libs import create_store_key
+pro = protocol.AhnlichProtocol(address="127.0.0.1", port=1369)
+client = client.AhnlichProtocol(client=pro)
+
+store_key = create_store_key(data=[5.0, 3.0, 4.0, 3.9, 4.9])
+
+
+response = client.delete_key(
+    store_name = "test store",
+    keys=[store_key]
+)
+```
+
+### Delete Predicate
+```py
+from ahnlich_client_py import AhnlichProtocol, AhnlichProtocol, query
+pro = protocol.AhnlichProtocol(address="127.0.0.1", port=1369)
+client = client.AhnlichProtocol(client=pro)
+
+condition = query.PredicateCondition__Value(
+                query.Predicate__Equals(key="job", value="sorcerer")
+            )
+
+
+response = client.delete_predicate(
+    store_name = "test store",
+    condition = condition
+)
+```
+
+
+
+## Bulk Requests
+The client has the ability to send multiple requests at once, and this requests will be handled sequentially. The builder class takes care of this. The response is a list of all individual request responses.
+
+
+```py
+from ahnlich_client_py.builders import AhnlichDBRequestBuilder
+from ahnlich_client_py import AhnlichProtocol, AhnlichProtocol, query
+pro = protocol.AhnlichProtocol(address="127.0.0.1", port=1369)
+client = client.AhnlichProtocol(client=pro)
+
+request_builder = AhnlichDBRequestBuilder()
+request_builder.ping()
+request_builder.info_server()
+request_builder.list_clients()
+request_builder.list_stores()
+
+response: server_response.ServerResult = request_builder.execute_requests(
+    client=client
+)
+
+```
+
+
+
+## Deploy to Artifactory
+
+Replace the contents of `MSG_TAG` file with your new tag message
+
+From Feature branch, either use the makefile :
+```bash
+make bump-python-version [major, minor, patch] 
+```
+or
+```bash
+poetry run bumpversion [major, minor, patch] 
+```
+
+When Your PR is made, changes in the client version file would trigger a release build to Pypi
+
+NOTE:
+1. `current_version` in `.bumpversion.cfg` and `CLIENT_VERSION` should be equal to the last/current version of a package (it will be upgraded correspondingly to the release by user's choice (`major, minor, patch`))
```

Comment:
not sure what CLIENT_VERSION means tbh... client refers to python library or go mod or whatever when I think what you are trying to represent is the types/server/network version correct?

Reply:
CLient version represents the version of the client itself. It's separate from the types crate version.
Whenever there's a change in the client that has nothing to do with the types crate, this is what's bumped up.
`make bump-python-version`. Look at deploy to artifactory